onnx/test/reference_evaluator_ml_test.py

```diff
@@ -837,14 +838,17 @@ def test_tree_ensemble_regressor(self):
             "MIN": np.array([[0.076923], [0.076923], [0.076923]], dtype=np.float32),
             "MAX": np.array([[0.5], [0.5], [0.5]], dtype=np.float32),
         }
-        for agg in ("SUM", "AVERAGE", "MIN", "MAX"):
-            expected = expected_agg[agg]
-            with self.subTest(aggregate_function=agg):
-                onx = self._get_test_tree_ensemble_regressor(agg)
-                self._check_ort(onx, {"X": x}, equal=True)
-                sess = ReferenceEvaluator(onx)
-                got = sess.run(None, {"X": x})
-                assert_allclose(expected, got[0], atol=1e-6)
+        for base_value in (None, [1.0]):
```

Comment:
I would parameterize this test using parameterized.parameterized.expand

Reply:
@justinchuby I have gone ahead and switched the test to use parameterized arguments. Please take a look.